app/scripts/controllers/newfromtemplate.js

```diff
@@ -355,6 +354,13 @@ angular.module('openshiftConsole')
             $scope.parameterDisplayNames[parameter.name] = parameter.displayName || parameter.name;
           });
 
+          _.each($scope.template.parameters, function(parameter) {
+            var found = _.find(JSON.parse($routeParams.templateParamsMap), function(param){
```

Comment:
Need to catch the exception on JSON parse failures. This is not JSON we're generating, so it might have syntax errors.